rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -2375,6 +2375,35 @@ public UserData getCurrentUserData() {
         return userData;
     }
 
+    @Override
+    public void releaseDanglingBusyNodes(List<NodeSet> busyNodesWithTask) {
```

Comment:
Why not call this method simply releaseNodes ?

Reply:
I am not sure of the new logic that must be introduced here, over the existing releaseNodes method. Cannot the existing method be reused or slightly modified to satisfy all needs ?

Reply:
This method releases the busy nodes that are *not* present in the given list. So it could be named like `releaseBusyNodesNotIn` for example. What do you think?

Reply:
Renamed into `releaseBusyNodesNotIn(List<NodeSet> nodesToNotRelease)`